pkg/api/customization/cluster/actionhandler.go

```diff
@@ -525,3 +531,56 @@ func (a ActionHandler) RotateCertificates(actionName string, action *types.Actio
 	http.ServeContent(apiContext.Response, apiContext.Request, "rotate", time.Now(), reader)
 	return nil
 }
+
+func (a ActionHandler) BackupEtcdHandler(actionName string, action *types.Action, apiContext *types.APIContext) error {
+	cluster, err := a.ClusterClient.Get(apiContext.ID, metav1.GetOptions{})
```

Comment:
ideally we should validate access first, similar to https://github.com/rancher/rancher/blob/486b190eb52ae8b5b7ca1cec840fab39b44202c8/pkg/api/customization/cluster/actionhandler.go#L495